v5/pivotal/projects.go

```diff
@@ -116,3 +116,39 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
@dav I think that to keep things consistent we should put this under `StoryService.Create`. I am follow the same convention (more or less) as go-github, so you can check e.g. https://godoc.org/github.com/google/go-github/github#IssuesService.Create